### ISSUE TITLE
Blazor WebAssemblyPrerendered load images twice

### ISSUE THREAD

**Issue by welisonmenezes** [dotnet/aspnetcore#40978]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

We have an App running with Server Side Rendering (WebAssemblyPrerendered). Our _Host.cshtml looks like below:

![image](https://user-images.githubusercontent.com/5102653/161118330-67c91f2c-1481-431a-834c-0ecfb4777088.png)

The problem is that the images are being requested twice.

![image](https://user-images.githubusercontent.com/5102653/161118250-3ce2d042-18e0-4230-b73c-002cc3effa83.png)

### Expected Behavior

We have to keep the server side content already loaded for SEO reasons, but we do not want to get assets twice.

### .NET Version

6.0.201

**Comment by javiercn** [comment]
@welisonmenezes thanks for contacting us.

Can you trace who's requesting each file twice? I suspect is the service worker you have in the page installing the app. I would suggest you add caching headers to your images to prevent the multiple requests hitting the server.

**Comment by welisonmenezes** [comment]
@javiercn thanks for your answer.

I am running the application locally for while. I've just noticed the behavior that I mencioned before. It looks the images are loaded from server side rendering before, and after the blazor wasm is ready on the client side the images are requested again.

![image](https://user-images.githubusercontent.com/5102653/161121695-bb322d75-6db6-452c-bbe8-c0983879f52e.png)
